WooCommerce/Classes/ViewRelated/Dashboard/Factories/StatsDataTextFormatter.swift

```diff
@@ -279,4 +279,18 @@ private extension StatsDataTextFormatter {
     enum Constants {
         static let placeholderText = "-"
     }
+
+
+}
+
+private extension Decimal {
+    func rounded(to scale: Int16) -> Self {
+        let numberHandler = NSDecimalNumberHandler(roundingMode: .plain,
+                                                   scale: scale,
+                                                   raiseOnExactness: false,
+                                                   raiseOnOverflow: false,
+                                                   raiseOnUnderflow: false,
+                                                   raiseOnDivideByZero: false)
+        return (self as NSDecimalNumber).rounding(accordingToBehavior: numberHandler) as Decimal
+    }
```

Comment:
We already have a private rounding method in `Decimal+Helpers`:

https://github.com/woocommerce/woocommerce-ios/blob/f1943c00c19f3d62e4d6b5a7d6256d211750a0fd/WooCommerce/Classes/Extensions/Decimal%2BHelpers.swift#L11-L16

What do you think about reusing that method rather than adding a new one here?

Reply:
Switched to it in ca731ef.